README.md

```diff
@@ -7,16 +7,8 @@ This plugin provides client-side integration for the [CodePush service](http://c
 * [Supported Components](#supported-components)
 * [Getting Started](#getting-started)
 * [iOS Setup](#ios-setup)
-    * [Plugin Installation](#plugin-installation-ios)
```

Comment:
Probably to clean up unnecessary links, and noise in index. I see windows setup section added below which would need adding these sublinks under that header too in the index. I am fine either ways.